go/vt/vtgate/planbuilder/plan_test.go

```diff
@@ -590,19 +590,16 @@ func loadSchema(t testing.TB, filename string, setCollation bool) *vindexes.VSch
 		t.Fatal(err)
 	}
 	vschema := vindexes.BuildVSchema(formal, sqlparser.NewTestParser())
-	if err != nil {
```

Comment:
One last `t.Fatal` left in this function on line 590. Mind fixing that while you are doing all the rest?